casbin/examples/MoreParam.conf

```diff
@@ -0,0 +1,11 @@
+[request_definition]

```

Comment:
These are you innovations right? good job, you should mention it at doc so others know the meaning?